src/univariate/discrete/dirac.jl

```diff
@@ -0,0 +1,30 @@
+"""
+    Dirac(value)
+
+A *Dirac distribution* is parametrized by its only value, and takes its value with probability 1.
+
+```julia
+d = Dirac(2.0)  # Dirac distribution with value = 2.
+rand(d)         # Always returns the same value
+```
+"""
+struct Dirac{T} <: DiscreteUnivariateDistribution
```

Comment:
Dirac Delta is a Continuous distribution

Reply:
No, it doesn't have a density with respect to the Lebesgue measure, so it is not a "continuous probability distribution" (not to mix with distributions in Schwartz' sense.)

Reply:
According to the documentation `ValueSupport` can take values `Discrete` (Samples take discrete `Int` values) and `Continuous` (samples take continuous real `Float64` values). So in this sense, it seems that the Dirac delta should be `Continuous`. I think that your interpretation would leave many useful measures completely out of the hierarchy (e.g. mixtures of continuous and discrete such as Spike-and-Slab)

Reply:
That is, according to the documentation the trait "Continuous" seems to refer only to the set on which the distribution is defined (the real line), not to the fact that it is absolutely continuous with respect to the Lebesgue measure.